Warn about sourcemap leakage (#10637)
getsentry/sentry-docs: docs/platforms/javascript/common/sourcemaps/uploading/uglifyjs.mdx

  - javascript.vue
---

In this guide, you'll learn how to successfully upload source maps for SystemJS using our `sentry-cli` tool.

<Include name="debug-id-requirement.mdx" />

### 1. Generate Source Maps

First, configure UglifyJS to output source maps:

```
uglifyjs app.js \
  -o app.min.js.map \
  --source-map url=app.min.js.map,includeSources
```

> <Alert>
>   Generating sourcemaps may expose them to the public, potentially causing your
>   source code to be leaked. You can prevent your source maps from being publicly
>   accessible by configuring your server to deny access to `.js.map` files, or by
>   deleting the sourcemaps before deploying your application.
> </Alert>

Reviewer: ```suggestion
<Alert>
Generating sourcemaps may expose them to the public, potentially causing your source code to be leaked. You can prevent this by configuring your server to deny access to `.js.map` files, or by deleting the sourcemaps before deploying your application.
</Alert>
```
